### PR TITLE
Add -, _, and * to valid characters to prefix and suffix a uwu

### PR DESCRIPTION
**Pull request by mja00**
This will let me `_uwu-`

### CODE REVIEW

**Comment by Chew** [comment]
why

**Comment by mja00** [comment]
Because I want to

**Review by Chew — DISMISSED** [review]

File: src/main/java/pw/chew/chanserv/listeners/UwUChannelHandler.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import net.dv8tion.jda.api.hooks.ListenerAdapter;
 
 public class UwUChannelHandler extends ListenerAdapter {
-    private final String UWU_REGEX = "[\\.|?|!|~]";
+    private final String UWU_REGEX = "[\\.|?|!|~|\\-|_|\\*]";
```
Comment (Chew) [inline src/main/java/pw/chew/chanserv/listeners/UwUChannelHandler.java]:
```suggestion
    private final String UWU_REGEX = "[\\.|?|!|~|\\-|_|*]";
```

Comment (mja00) [reply src/main/java/pw/chew/chanserv/listeners/UwUChannelHandler.java:8]: We need to escape the * character since that is a valid regex character

Comment (mja00) [reply src/main/java/pw/chew/chanserv/listeners/UwUChannelHandler.java]: Testing with regex101 shows it needs to be 
```suggestion
    private final String UWU_REGEX = "[.?!~\-_*]";
```
The `|` are redundant in a `[]`

File: src/main/java/pw/chew/chanserv/listeners/UwUChannelHandler.java [hunk 2]
```diff
@@ -5,7 +5,7 @@
 import net.dv8tion.jda.api.hooks.ListenerAdapter;
 
 public class UwUChannelHandler extends ListenerAdapter {
-    private final String UWU_REGEX = "[\\.|?|!|~]";
+    private final String UWU_REGEX = "[.?!~\-_*]";
```
Comment (Chew) [inline src/main/java/pw/chew/chanserv/listeners/UwUChannelHandler.java]:
```suggestion
    private final String UWU_REGEX = "[.?!~\\-_*]";
```

**Review by Chew — APPROVED** [review]

**Review by Chew — APPROVED** [review]

**Pull request merged**